ResearchKit/ResearchKit_Private.h

```diff
@@ -114,5 +115,7 @@
 #import <ResearchKit/ORKTouchRecorder.h>
 #import <ResearchKit/ORKHTMLPDFPageRenderer.h>
 
+#import <ResearchKit/ORKEnvironmentSPLMeterResult.h>
+
```

Comment:
Can we remove this once the visibility for this header has been changed to public?